nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,70 @@
+const express = require("express");
```

Comment:
Nitpick(readability):
Using whitespace can make code easier to read by making it more clear where one piece starts and ends.

Consider breaking your file into blocks, maybe something like this (without the comments):
```
// All imports
const express = require("express");
const meals = require("./data/meals");

// All global variables
const app = express();
const mealsNum = meals.map((meal) => meal.id);

// One block per route
app.get("/", async (request, response) => {
  ...
});

app.get("/meals", async (request, response) => {
  ...
});

// Exports are usually put at the end
module.exports = app;
```

Reply:
Good to hear that, so clear code. thanks for telling. :)